v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/reservations_client.py

```diff
@@ -341,12 +348,7 @@ async def __poll_dav_calendar(self, kwargs=None):
             )
             remote_v2g_events.append(v2g_event)
 
-        attributes = {"keep_alive": start}
-        await self.hass.set_state(
-            "sensor.calendar_account_connection_status",
-            state="Successfully connected",
-            attributes=attributes,
-        )
+        await self.v2g_main_app.set_caldav_connection_status(connected=True)
```

Comment:
Replacing a event based notification with a direct method call to the v2g_main_app does not "loosen" the connection from reservations_client to v2g_liberty, which is highly desirable.
Instead, we should try to get reservations_client (and other modules) as unaware as possible from v2g_liberty.

Reply:
I agree. I was working towards an other goal: the reservations_client should not know about HA entities. The de-coupling will be fixed with the introduction of pyee.
What should get priority in your opinion?

Reply:
Why do you think the reservation client should not know about HA entities? Especially the HA integration calendar is using HA entities anyway.

Reply:
I think adding more calls back to other modules is a regression, so I would prefer replacing it with an event interface instead.

Reply:
OK, will keep it as is for now awaiting the pyee implementation.